updates all the incorrect tocs AND the maintenance instructions (#216)
OHS-Hosting-Infrastructure/infrastructure: docs/runbooks/maintenance.md

Common things to note/do during maintenance.

## Table of Contents

<!-- toc -->

* [Create a new volume](#create-a-new-volume)
* [Attach a volume to an EC2 instance](#attach-a-volume-to-an-ec2-instance)
* [Add Disk Space](#add-disk-space)
* [Examine and partition a new disk](#examine-and-partition-a-new-disk)
* [Mount a disk](#mount-a-disk)
* [Unmount a disk](#unmount-a-disk)
* [Add a label to a disk](#add-a-label-to-a-disk)
* [Sync the backup](#sync-the-backup)
* [Update NodeJS to version 12](#update-nodejs-to-version-12)
* [Updating packages on Ubuntu](#updating-packages-on-ubuntu)
* [Updating pacakages on Centos](#updating-pacakages-on-centos)
* [Cleaning up old kernels on Centos](#cleaning-up-old-kernels-on-centos)

<!-- Regenerate with "pre-commit run -a markdown-toc" -->

<!-- tocstop -->

> ## Create a new volume

Reviewer: Hah! I think something we both missed is when you create the new volume enter in a number greater than the size previously.

Also note you cannot downsize a volume from a snapshot.

1. Then [examine and partition the new disk](#examine-and-partition-a-new-disk)
1. [Mount the disk](#mount-a-disk)  (if unmounted) to determine whether you need to add a filesystem. 

   1. If you have an issue with the file type, you can repair by running the following on an unmounted disk: `[sudo] xfs_repair /dev/<DEVICE_ID><partition>`. (You may have to run `[sudo] xfs_repair -L /dev/<DEVICE_ID><partition>` instead).
   1. If the mounted disk does not show the correct size after you mount it (examine by running `df -h`), run `[sudo] xfs_growfs /var/<MOUNT>`. Run `df -h` again to confirm it is the correct size.

1. Add a `ext4` filesystem on top. Note, you may want an xfs file system but that will use different utlities.
   1. use `mkfs.ext4` on the partition device you created.
1. [Add a label to a disk](#add-a-label-to-a-disk) to give it a human-readable identifier.
1. If you're switching a volume for a copy of a larger size:
   1. [Sync the backup](#sync-a-backup) then [unmount](#unmount-a-disk) both the current volume and replacement volume. 

>    1. Then [mount the replacement volume](#mount-a-disk) to the original disk. XXXXXXXXX THIS PART. 

Reviewer: did you mean to delete or replace this line?

Author: No--I wrote a note about it in the PR:

PLEASE HELP!
There's a line (https://github.com/OHS-Hosting-Infrastructure/infrastructure/compare/rek-131-increase-disk-space?expand=1#diff-85e9d0513e5021b6c54bd48397c4a366d3498e677637372ab2014a41b78b737fR58) marked "XXXXXXXXX THIS PART. " where I am still a little confused what's happening. Can you check that what I've written is accurate and fully represents the action needed at that part? I'm still a little confused why we sync to the second disk (the temp one) and then mount to the first disk anyway (the preexisting one).

Reviewer: Got it! I'll take a look.

Reviewer: OOH I SEE... lemme suggest a change.

Reviewer: ```suggestion
   1. Then [mount the replacement volume](#mount-a-disk) to the original mount path in the filesystem.
```
